test/parallel/test-zlib-const.js

```diff
@@ -4,14 +4,35 @@ const assert = require('assert');
 
 const zlib = require('zlib');
 
-assert.strictEqual(zlib.constants.Z_OK, 0, 'Z_OK should be 0');
+assert.strictEqual(zlib.constants.Z_OK, 0,
+                   [
+                     'Expected Z_OK to be 0;',
+                     `got ${zlib.constants.Z_OK}`
+                   ].join(' '));
```

Comment:
I'm not a fan of this formatting, but I had to adhere to linter rules; specifically, max line length of 80.  I would prefer format like this:

```
assert.strictEqual(zlib.codes.Z_OK, 0,
          `Z_OK should be immutable. Expected to get 0, got ${zlib.codes.Z_OK}`);
```

What would folks think about overriding the max-line-length linter rule to 120 characters for this file?

Reply:
That is what is done in `test/parallel/test-require-extensions-same-filename-as-dir-trailing-slash.js` by adding `/* eslint-disable max-len */`.

Reply:
Ah, nice. That's what I had in mind to do. Guess I didn't want to rock the linting rules boat on my first contribution to Node, eh? Ha! Will override here.

Reply:
This would also work
```js
assert.strictEqual(
  zlib.constants.Z_OK,
  0,
  `Expected Z_OK to be 0; got ${zlib.constants.Z_OK}`
);
```

Deactiviating the max-len is not a good idea.